op/op.go

```diff
@@ -52,12 +52,39 @@ func New(id string, result map[string]any, status Status, err error, params map[
 
 // StatusCounts takes a slice of op references and returns a map containing sums of each Status
 func StatusCounts(ops map[string]Op) (map[Status]int, error) {
-	statuses := make(map[Status]int)
-	for _, o := range ops {
-		if o.Status == "" {
-			return nil, fmt.Errorf("unable to build Statuses map, op not run: op=%s", o.Identifier)
-		}
-		statuses[o.Status]++
+	// copy our input into a new map that conforms to our Walk input type
+	m := make(map[string]any, len(ops))
+	for k, v := range ops {
+		m[k] = v
+	}
+
+	// Create our accumulator and Walk ops
+	acc := make(map[Status]int, 0)
```

Comment:
I'm unclear on why this map would be initialized with a size of 0. 🤔 Would it be better to omit the size or even to declare `var acc map[Status]int` instead?

Reply:
Totally right; thanks for catching that!

Reply:
I changed this to initialize to len(ops), which is the minimum length this will have (if there are no nested ops).